.repo-metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "@google-cloud/bigquery",
+    "name_pretty": "Google Cloud Bigquery",
+    "product_documentation": "https://cloud.google.com/bigquery",
+    "client_documentation": "https://cloud.google.com/nodejs/docs/reference/bigquery/latest/",
+    "release_level": "ga",
+    "language": "nodejs",
+    "repo": "googleapis/nodejs-bigquery",
+    "distribution_name": "@google-cloud/bigquery",
+    "requires_billing": true
```

Comment:
@tswast @shollyman I know BigQuery has a generous free tier.  Does it require a billing account to get it?

Reply:
BigQuery sandbox no longer requires a billing instrument to be associated with the project. Background info at https://cloud.google.com/blog/products/data-analytics/query-without-a-credit-card-introducing-bigquery-sandbox

Reply:
Does the sandbox only pertain to the web-ui? I was thinking they still have to enter billing info if they're going to use the client library, so it would be true in this documentation. Should requires_billing still be set to false anyway, to reflect that the product itself has non-billing options?

Reply:
Programmatics access also counts under the free tier.

Reply:
Okay, will update! Thank you!